src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +66,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      */
     public List<People> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() >= fromAge
+                        && (p.getSex() == People.Sex.WOMEN && p.getAge() <= femaleToAge)
+                        | (p.getSex() == People.Sex.MAN && p.getAge() <= maleToAge))
```

Comment:
```suggestion
                        || (p.getSex() == People.Sex.MAN && p.getAge() <= maleToAge))
```
this called short version of OR operator))